pkg/features/kube_features.go

```diff
@@ -628,6 +628,7 @@ const (
 	// owner: @HirazawaUi
 	// kep: https://kep.k8s.io/4369
 	// alpha: v1.30
+	// beta: v1.32
```

Comment:
please remove L630-L631. see https://github.com/kubernetes/kubernetes/issues/127782 for the reason.